client/state/automated-transfer/actions.js

```diff
@@ -4,22 +4,36 @@
 import {
 	AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST,
 	AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE,
+	AUTOMATED_TRANSFER_STATUS_REQUEST,
 	AUTOMATED_TRANSFER_STATUS_SET,
 } from 'state/action-types';
 
 /**
+ * Query the automated transfer status of a given site.
+ *
+ * @param {number} siteId The id of the site to query.
+ * @returns {Object} An action object
+ */
+export const getAutomatedTransferStatus = ( siteId ) => ( {
+	type: AUTOMATED_TRANSFER_STATUS_REQUEST,
+	siteId,
+} );
+
+/**
  * Sets the status of an automated transfer for a particular site
  *
  * @see state/automated-transfer/constants#transferStates
  *
  * @param {number} siteId The site id to which the status belongs
  * @param {string} status The new status of the automated transfer
+ * @param {string} uploadedPluginId Id of any uploaded plugin
  * @returns {Object} An action object
  */
-export const setAutomatedTransferStatus = ( siteId, status ) => ( {
+export const setAutomatedTransferStatus = ( siteId, status, uploadedPluginId ) => ( {
```

Comment:
Unless I'm missing anything here, it seems we'll be able to set the status by either specifying a particular status, or by checking the status of a plugin upload. For clarity I'd suggest documenting how `uploadedPluginId` works and that we could use either it or `status` to set the AT transfer status.